ci-operator/step-registry/cucushift/installer/rehearse/vsphere/upi/disconnecting-network/provision/cucushift-installer-rehearse-vsphere-upi-disconnecting-network-provision-chain.yaml

```diff
@@ -0,0 +1,26 @@
+chain:
+  as: cucushift-installer-rehearse-vsphere-upi-disconnecting-network-provision
+  steps:
+  - ref: ipi-install-rbac
+  - ref: openshift-cluster-bot-rbac
+  - ref: ipi-conf
+  - ref: ipi-conf-telemetry
+  - ref: ipi-conf-vsphere-check
+  - chain: vsphere-provision-bastionhost
+  - ref: vsphere-provision-bastionhost-dnsmasq
+  - ref: vsphere-disconnecting-network-dns-update
```

Comment:
Because the step actually is to replace the default dns to use dnsmasq server, so not involve "disconnecting-network" yet, so maybe we can move it to some other folder.

Reply:
How about `vsphere-provision-bastionhost-dnsmasq-dns-update`?  or even we can merge `vsphere-disconnecting-network-dns-updat` to `vsphere-provision-bastionhost-dnsmasq`?

Reply:
I think the latter is better, let me merge `vsphere-disconnecting-network-dns-update` to `vsphere-provision-bastionhost-dnsmasq`.

Reply:
updated.